sdk/storage/azure-storage-file-share/azure/storage/fileshare/_share_service_client.py

```diff
@@ -93,7 +93,7 @@ def __init__(
         self._query_str, credential = self._format_query_string(sas_token, credential)
         super(ShareServiceClient, self).__init__(parsed_url, service='file-share', credential=credential, **kwargs)
         self._client = AzureFileStorage(version=VERSION, url=self.url, pipeline=self._pipeline)
```

Comment:
Why is it necessary to add the next line? can we assign `api_version`  to version here?